### DISCUSSION TITLE
Does Zustand work for React < 17 applications?

### DISCUSSION THREAD

**Discussion by matheuspalmawm** [pmndrs/zustand#2425] [Q&A] [answered]
I was trying to use Zustand along with a React < 17 application and I was not able to do it, and I also I found nothing about it

**Comment by dbritto-dev** [comment, marked as answer]
@matheuspalmawm zustand supports react 16.8 and onwards

**Reply by matheuspalmawm** [reply]
Thanks for the quick response, good to know!, I'm trying to build a external library to share the State Management Logic with other applications

**Reply by dbritto-dev** [reply]
@matheuspalmawm btw, in any case we recommend to use the latest stable version
